technical-specification/sections/zz-references.adoc

```diff
@@ -0,0 +1,16 @@
+
+[bibliography]
+== Bibliography
+
+* [[[isotr17755,ISO/TR 17755:2014]]] _ISO/TR 17755, Fire safety -- Overview of national fire statistics practices_
+
+* [[[iso13943,ISO 13943:2017]]]	_ISO 13943:2017, Fire Safety -- Vocabulary_
```

Comment:
There's some spacing issue (tab?) here, can you help fix?

It should be:
```
* [[[iso13943,ISO 13943:2017]]], _ISO 13943:2017, Fire Safety -- Vocabulary_
```

Reply:
Huh, my text editor wasn't showing me that up. Fixed.

Reply:
I added the missing comma to other references as well.